.travis.yml

```diff
@@ -1,54 +1,17 @@
 language: dart
 
-# Workaround for issue with sandboxed Chrome in containerized Travis builds.
-sudo: required
-
-addons:
-  apt:
-    packages:
-    - ttf-kochi-mincho
-    - ttf-kochi-gothic
-    - ttf-dejavu
-    - ttf-indic-fonts
-    - fonts-tlwg-garuda
-  chrome: stable
-
 dart:
-  - 1.24.3
+  - 2.3.0
   - stable
+  - dev
 
-before_install:
-  # Workaround for https://github.com/travis-ci/travis-ci/issues/8607
-  - sudo rm -vf /etc/apt/sources.list.d/*riak*
-  # Content shell needs this font. Since it has a EULA, we need to manually
-  # install it.
-  #
-  # TODO: remove this and use "sudo: false" when travis-ci/travis-ci#4714 is
-  # fixed.
-  - sudo apt-get update -yq
-  - sudo sh -c "echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections"
-  - sudo apt-get install msttcorefonts -qq
-
-  - mkdir -p bin
-  - export PATH="$PATH:`pwd`/bin/"
-  - ln -s `which chromium-browser` bin/google-chrome
-
-  - wget "http://gsdview.appspot.com/dart-archive/channels/stable/release/latest/dartium/content_shell-linux-x64-release.zip"
-  - unzip content_shell-linux-x64-release.zip
-  - ln -s `pwd`/`echo drt-linux-*`/content_shell bin/content_shell
-
-  -  export DISPLAY=:99.0
-  -  ./tool/setup_dartium.sh
-  -  export PATH=$PATH":$PWD"
-  -  sh -e /etc/init.d/xvfb start
+sudo: required
+addons:
+  chrome: stable
 
 script:
-  - pub run dependency_validator -i browser,coverage,dart_style,test
-  - pub run dart_dev dart2-only -- format --check
+  - pub run dependency_validator -i pedantic
```

Comment:
Do you think it would be worth making a standard `DependencyValidatorTool`?

Reply:
I had talked with Rob about supporting config via yaml (either in something like a `dependency_validator.yaml` or via a section in `pubspec.yaml`), so that you could always run `pub run dependency_validator` without any args. The main advantage of that approach is to not tie dep validator to dart_dev semantics, but I'm open to being convinced otherwise

Reply:
Yeah, either approach would be fine to me.